src/openeo_test_suite/tests/general/payload_examples/post_jobs/post_jobs_1.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
How does this ensure the data and file formats are available?

Reply:
It does not, is there some base dataset/file format that we can expect to always be here?

Is there a way we can include a baseline/test process that can be guaranteed to never fail?

Reply:
I'd just do the same as in process testing: skip if not available with a corresponding message.